code/game/machinery/deployable.dm

```diff
@@ -21,14 +21,22 @@
 	var/debris_type
 
 
-/obj/structure/barricade/proc/take_damage(damage, leave_debris=1, message)
+/obj/structure/barricade/proc/take_damage(damage, damage_type = BRUTE, sound_effect = 1)
```

Comment:
standardization of take_damage() proc. All versions use the same arguments.
Maybe if I have time later, I can Move the proc to obj/structure/proc/take_damage() , like I already did for machines.